projects/1.18/assets/mob-battle-mod/mobbattle/lang/zh_cn.json

```
@@ -1 +1,56 @@
-{}
\ No newline at end of file
+{
+  "item.mobbattle.mob_stick": "生物激怒杖",
+  "item.mobbattle.mob_kill": "生物击杀杖",
+  "item.mobbattle.mob_heal": "生物治疗杖",
+  "item.mobbattle.mob_group": "生物激怒杖（多选）",
+  "item.mobbattle.mob_effect": "状态去除杖",
+  "item.mobbattle.mob_effect_give": "状态给予杖",
+  "item.mobbattle.mob_armor": "盔甲编辑杖",
+  "item.mobbattle.mob_army": "生物组队杖",
+  "item.mobbattle.mob_mount": "生物骑乘杖",
+  "item.mobbattle.mob_equip": "生物拾取物品杖",
+  "item.mobbattle.egg_ex": "队伍内刷怪蛋",
+  "itemGroup.mobbattle.tab": "怪物大乱斗",
+  "conf.mobbattle.particle": "显示队伍粒子效果",
+  "conf.mobbattle.addai": "自动添加队伍目标AI",
+  "tooltip.spawnegg": "左击实体以存进刷怪蛋，按住潜行可以保存该实体 NBT 数据",
+  "tooltip.spawnegg.spawn": "生成 %s",
+  "tooltip.spawnegg.save": "已保存实体 %s",
+  "tooltip.spawnegg.creative": "仅在创造模式下可复制实体",
+  "tooltip.armor": "右击实体以编辑他的盔甲和物品栏",
+  "tooltip.army.first": "右击方块设定框选的第一个对角线角落方块，再次右击其他方块设定框选的第二个对角线角落方块以完成框选（类似 WorldEdit）",
+  "tooltip.army.second": "右击空气将框选内所有实体纳入已存在的同名团队（如果团队不存在则为默认团队）",
+  "tooltip.army.third": "潜行并右击以重设框选",
+  "tooltip.army.forth": "左击实体以将其纳入已存在的同名团队（如果团队不存在则为默认团队）",
+  "tooltip.army.fifth": "使用原版 /team 命令以查看或更改团队颜色",
+  "tooltip.army.add": "已将实体纳入团队 %s",
+  "tooltip.army.add.box": "已将框选内所有实体纳入团队 %s",
+  "tooltip.army.reset": "已重置框选角落方块",
+  "tooltip.effect.remove": "左击实体以移除它的状态效果",
+  "tooltip.effect.remove.clear": "效果已移除",
+  "tooltip.effect.give.first": "左击实体以移除它的状态效果",
+  "tooltip.effect.give.second": "左击实体以移除它的状态效果",
+  "tooltip.effect.give.add": "已添加持续 %3$s tick 的 %2$s 倍率效果 %1$s",
+  "tooltip.equip.first": "右击方块设定框选的第一个对角线角落方块，再次右击其他方块设定框选的第二个对角线角落方块以完成框选（类似 WorldEdit）",
+  "tooltip.equip.second": "右击空气将使框选内的所有实体可以捡起掉落物",
+  "tooltip.equip.third": "潜行并右击以重设框选",
+  "tooltip.equip.box.add": "框选内所有实体现在都可以捡起掉落物",
+  "tooltip.equip.add": "实体现在可以捡起掉落物",
+  "tooltip.equip.reset": "已重置框选角落方块",
+  "tooltip.group.first": "左击以多选实体",
+  "tooltip.group.second": "右击实体以设置为目标",
+  "tooltip.group.third": "潜行并右击以重置",
+  "tooltip.group.remove": "已移除实体",
+  "tooltip.group.reset": "已重置所有实体",
+  "tooltip.group.add": "已添加实体",
+  "tooltip.heal": "左击实体以治疗它",
+  "tooltip.kill": "左击实体以杀死它",
+  "tooltip.mount.first": "第一次左击实体以选择被骑乘实体",
+  "tooltip.mount.second": "第二次左击其他实体将会让它骑在被骑乘实体上",
+  "tooltip.mount.reset": "已重设选择实体",
+  "tooltip.stick.contains": "已选择实体：%s",
+  "tooltip.stick.first": "左击以选择实体",
+  "tooltip.stick.second": "右击以重置选择实体",
+  "tooltip.stick.reset": "已重置选择实体",
+  "tooltip.stick.add": "已选择实体，请选择其他实体作为其攻击目标"
```

```suggestion
  "tooltip.stick.add": "首个实体已选择，请选择其他实体作为其攻击目标"
```

---

```suggestion
  "tooltip.stick.add": "已选择第一个实体，请选择第二个实体作为其攻击目标"
```
这个杖只能选两个实体让他们互殴，不能选>2个（所以有多选版本和团队）